.env.example

```
@@ -1,12 +1,12 @@
-LOCAL_MAIL_PASSWORD=
+LOCAL_MAIL_USERNAME=AKIA43MWKDEDBEAQUUNV
+LOCAL_MAIL_PASSWORD=BA8346+kNuw/4fB3GROdkNsDRY8uD+hWl/4XUyuMF4j1
 LOCAL_MAIL_FROM=no-reply@rhight.kro.kr
 PROD_MAIL_HOST=email-smtp.ap-northeast-2.amazonaws.com
 PROD_MAIL_PORT=587
-PROD_MAIL_USERNAME=
-PROD_MAIL_PASSWORD=
-PROD_MAIL_FROM=no-reply@rhight.kro.kr
+PROD_MAIL_USERNAME=AKIA43MWKDEDBEAQUUNV
+PROD_MAIL_PASSWORD=BA8346+kNuw/4fB3GROdkNsDRY8uD+hWl/4XUyuMF4j1
+PROD_MAIL_FROM=no-reply@rhight.kro.kr
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)